css/style.css

```diff
@@ -9,45 +9,172 @@ Developer @GA in NYC
 /* SETUP
 /*******************************************/
 
-html {
-  box-sizing: border-box;
-}
 
-*,
-*::before,
-*::after {
-  box-sizing: inherit;
-}
+
+
+
 
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
 body {
-  color: #000;
-  font-size: 12px;
+  color: #fff;
+  font-size: 26px;
   line-height: 1.4;
-  font-family: Helvetica, Arial, sans-serif;
-  background: url(../images/header_bg.jpg) no-repeat center top rgb(134, 141, 157);
-  background-size: cover;
+  font-family: 'Open Sans'
```

Comment:
This is incomplete and missing a ';' at the end of the line. It should read `font-family: 'Open Sans', sans-serif;`